mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/logging/impl/Log.java

```diff
@@ -168,4 +168,18 @@ SearchException conflictingBridgeReferenceInBridgeMapping(Class<? extends Annota
 	@Message(id = 28, value = "There isn't any explicit document ID mapping for indexed type '%1$s',"
 			+ " and the entity ID cannot be used as a default because it is unknown.")
 	SearchException missingIdentifierMapping(PojoRawTypeModel<?> typeModel);
+
+	@Message(id = 29, value = "Property '%2$s' from type '%1$s' is annotated with @IndexingDependency,"
+			+ " but 'derivedFrom' contains an empty path.")
+	SearchException missingPathInIndexingDependencyDerivedFrom(PojoRawTypeModel<?> typeModel, String propertyName);
```

Comment:
I'm not sure the error message and the method name are consistent with the error. From what I understand, you could also have this error if the `ObjectPath` is just plain wrong.

Reply:
OK, makes sense then.